### PR TITLE
style: better verify email view

### PR DESCRIPTION
**Pull request by djaiss**
This PR bonifies the Verify email view by 

* having a better layout
* putting the header so user can actually log out
* add an illustration
* use the built-in success message in the bottom right part of the screen.

Screen Shot 2020-12-12 at 13 14 44

### CODE REVIEW

File: resources/lang/en/auth.php [hunk 1]
```diff
@@ -22,10 +22,10 @@
     'login_no_account' => 'You are not lucky to have an account yet?',
     'login_register' => 'Create your account now',
 
-    'confirmation_title' => 'Verify Your Email Address',
+    'confirmation_title' => 'Please verify your email address.',
     'confirmation_fresh' => 'A fresh verification link has been sent to your email address.',
     'confirmation_check' => 'Before proceeding, please check your email for a verification link.',
-    'confirmation_request_another' => 'If you did not received the verification email, you can request another here:',
+    'confirmation_request_another' => 'If you did not receive the verification email, you can request another using this big blue button below.',
```
Comment (asbiin) [inline resources/lang/en/auth.php:28]: I see that's a funny message, but what if we change the colors later on?

**Review by asbiin — APPROVED** [review]
Great!
Just a comment below:

**Comment by asbiin** [comment]
That's more a 'style' change than a 'chore'

File: resources/lang/en/auth.php (hunk 1 above)
Comment (djaiss) [reply resources/lang/en/auth.php]: You are definitely right. Fixing this.

**Pull request merged**